grpc-native/src/main/java/org/ballerinalang/net/grpc/callback/UnaryCallableUnitCallBack.java

```diff
@@ -82,49 +82,51 @@ public void notifySuccess(Object response) {
             }
             return;
         }
+
+        Object content;
+        BMap headerValues = null;
+        if (isContextRecordByValue(response)) {
+            content = ((BMap) response).get(StringUtils.fromString("content"));
+            headerValues = ((BMap) response).getMapValue(StringUtils.fromString("headers"));
```

Comment:
Shall we make the "content" and "headers" constants?

Reply:
Fixed in 23fbf72